docs/installation_guide.rst

```diff
@@ -268,7 +269,117 @@ data will be unavailable.
 Docker Compose
 ================
 
-If you prefer to use docker compose, a docker-compose.yml template is provided below for convienence.
+If you prefer to use docker compose, a docker-compose.yml template is provided below for convenience.
+
+Using docker over a public network
+----------------------------------
+
+Since rotki was designed to run over a public network it is :ref:`generally advised <docker warning>` to avoid
+directly exposing your rotki instance over public network.
+
+If you still want to access rotki over a public network it is suggested to use at least a proxy with
+basic authentication and an intermediate layer of security. With this setup anyone that visits your public domain
```

Comment:
isn't the proxy with basic auth the intermediate layer of security?

Reply:
/and/as/